producer/src/main/kotlin/streams/StreamsEventRouterConfiguration.kt

```diff
@@ -23,12 +26,14 @@ private object StreamsRoutingConfigurationConstants {
     const val ENABLED = "streams.source.enabled"
     const val SCHEMA_POLLING_INTERVAL = "streams.source.schema.polling.interval"
     const val PROCEDURES_ENABLED = "streams.procedures.enabled"
+    const val KEY_STRATEGY_SUFFIX = "key_strategy"
```

Comment:
Dobbiamo fare così perché il `.` è ammesso come carattere valido per i topic quindi potresti trovarti `mytopic.` come carattere valido
```suggestion
    const val KEY_STRATEGY_SUFFIX = ".key_strategy"
```